### ISSUE TITLE
Blackmagic ATEM Constellation 4K Talkback

### ISSUE THREAD

**Issue by a1whittaker** [bitfocus/companion-module-bmd-atem#367]
Is there a possibility to add BM ATEM Constellation 4K talk back into the ATEM module. Our switcher lives in a rack room and we want to use the built in talk back function to our camera ops.
